### DISCUSSION TITLE
Introductions!

### DISCUSSION THREAD

**Discussion by wkearn** [csdms/project#25] [General]
Hi everyone!

I thought it would be nice to have a place to introduce ourselves to the CSDMS community, and say a little bit about our interests and things we're working on. Feel free to continue this discussion with your own introduction!

I'm Will (he/him), and I'm a Research Software Engineer at the University of Potsdam in Germany, where I'm working on the next generation of [TopoToolbox](https://topotoolbox.github.io), a software suite for analyzing digital elevation models. I've been a Julia user for most of my career, though I've started to get more familiar with other languages and tools as I've been working on TopoToolbox. My background is in coastal and marine geomorphology, and while I mostly work on software development these days, I'm also very interested in the statistical modeling of sediment transport and techniques for model-data integration. I live in Berlin, and outside of work, I am trying to get better at drawing.

**Comment by samharrison7** [comment]
Hi all!

I'm Sam (he/him) and I'm an environmental (exposure) modeller at the UK Centre for Ecology & Hydrology, based in Lancaster, UK. Most of my working revolves around the risk assessment of chemicals in the environment, and for the past few years I have been working a lot on microplastics and pharmaceuticals. My main specialism is environmental exposure modelling - predicting the concentrations of chemicals that organisms in the environment are exposed to. This spans from small scale models of individual processes (such as the [fragmentation of plastics](https://github.com/microplastics-cluster/fragment-mnp)), right up to [geospatial models](https://github.com/nerc-ceh/nanofase) of exposure at catchment or broader scales.

I'm not an RSE by job title, but a fair amount of my work covers RSE-type things, and I love writing software. My main languages are Python and Fortran, though I dabble in others. My first language was probably PHP (unless HTML counts) and the language I'd like to learn next is Rust!

Oh, and a shameless plug: I have a website, if anyone wants to learn more about what I do! https://samharrison.science.

**Comment by thermok-SG** [comment]
Hi everyone,

I'm Suryodoy (he/him) and I'm a postdoctoral fellow at the University of Plymouth. I'm working on building a new component for Landlab that will model the triggering of coseismic landslides, and their runout. My background is in structural geology and tectonics, however, having done my PhD on the thermokinematic modelling of the central Nepal Himalaya, with some bits of associated landscape evolution modelling as well. The 2015 Gorkha earthquake happened in the area that I studied, so that got me into earthquakes and natural hazards.

I'm more of a dabbler in programming. I've been a MATLAB user most of my programming career, but I also know bits of C++ and Fortran (emphasis on the bits). I've shifted recently to Python, ever since I lost access to MATLAB after leaving the Uni; this is what got me really keen on open-source programming as well.

**Comment by dorchard** [comment]
Hi, I'm Dominic (he/him). I am an academic at the University of Cambridge and the University of Kent. I am a computer scientist with a background in programming language design, semantics, and implementation. At Cambridge, I co-direct the [Institute of Computing for Climate Science](https://iccs.cam.ac.uk/) where we study how to support climate, earth systems, and environmental modelling through developments in software engineering, computer science, data science, and machine learning. Through this, I spend time thinking about how to build programming languages, tools, and systems to support science. I continue to do quite a bit of work on programming language theory and related ideas embedded more in computer science. Overall, I love programming and doing maths. My website has links to places you can find me: https://dorchard.github.io/

**Comment by sporella** [comment]
Hi everyone, I'm Steph. I'm a Research Software Engineer / Data Scientist working at Cienciambeintal in Chile. My background is agricultural engineering. I worked in my early career and during my master at a climate change research center, and there I discovered the world of programming (and my love for it). Now I work developing data and software solutions in environmental research. I'm a volunteer at Py and R Ladies, organizations that promote women's participation and leadership in programming communities. Outside work I have a lot of interests but now I'm working in an edible garden in my patio, I like cooking and literature :)
